pom.xml

```diff
@@ -20,6 +20,46 @@
             <artifactId>jakarta.annotation-api</artifactId>
             <version>2.1.1</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
it is better to use JUnit 5: https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-engine

Also, ChatGPT is using JUnit 4 by default, but you may ask to generate code using JUnit 5